rdf-star-cg-spec.html

```diff
@@ -989,6 +1013,29 @@ <h2>SA-mode and PG-mode</h2>
       </ul>
     </section>
 
+    <section>
+      <h2>The seminal example</h2>
+
+      <p>The motivating example in the original RDF* paper [[RDF-STAR-FOUNDATION]] was on a provenance use-case, and is repeated below.</p>
+
+      <pre data-transform="updateExample"
+        data-content-type="text/x-turtle-star"
+        class="nohighlight example"
+      >
+        <!--
+        # the controversial seminal example
+        :bob foaf:name "Bob".
+        <<:bob foaf:age 23>> dct:creator <http://example.com/crawlers#c1> ;
+                             dct:source <http://example.net/listing.html> .
+        -->
+      </pre>
+
+      <p>This example was further debated on the <a href="https://lists.w3.org/Archives/Public/public-rdf-star/">RDF* mailing list</a>, as it may have set wrong expectations about what RDF* embedded triples represent. More precisely, from this example, one may wrongly assume that `<<:bob foaf:age 23>>` represent the <em>occurrence</em> of the given triple at the address `http://example.net/listing.html`. This impression may be reinforced by the use of `dct:creator`: arguably, a triple (as a unique abstract entity) is not "created" by anyone, while an occurrence thereof can be said to be created of authored.</p>
+
+      <p>In retrospect, it would probably have been better to only use the `dct:source` property on the triple, and maybe to give it two values (as it is possible for the same unique triple to be mentioned in several sources). Another solution would have been to adopt a more complex modeling as in <a href="#type-token-example"></a>. However, the purpose of that example was not to discuss the subtleties of the type-token distinction (see <a href="#type-token"></a>), but on the contrary to provide a simple minimal example of how RDF* could be used.</p>
```

Comment:
```suggestion
      <p>In retrospect, it would probably have been better to only use the `dct:source` property on the triple, and maybe to give it two values (as it is possible for the same unique triple to be mentioned in several sources). Another solution would have been to adopt a more complex modeling, as in <a href="#type-token-example"></a>. However, the purpose of that example was not to discuss the subtleties of the type-token distinction (see <a href="#type-token"></a>), but on the contrary to provide a simple minimal example of how RDF* could be used.</p>
```